conf/messages

```diff
@@ -98,6 +98,40 @@ setup.footer.copyright=&copy; 2015 yetu AG
 setup.footer.privacy=Privacy Policy
 setup.footer.imprint=Imprint
 setup.footer.terms=Terms &amp; conditions
+setup.next=next &gt;
+
+#setup registration page
+setup.registration.register=Register at yetu
+setup.registration.already=Already registered?
+setup.registration.hint=all fields are mandatory
+setup.registration.firstname=First name
+setup.registration.lastname=Last name
+setup.registration.email=E-mail
+setup.registration.emailplaceholder=this email will be your username
+setup.registration.password=Password
+setup.registration.passwordplaceholder=new password for yetu login
+setup.registration.repeatpassword=Repeat password
+setup.registration.repeatpasswordplaceholder=repeat new password for yetu login
+setup.registration.terms=I accept the
+setup.registration.termslink=terms &amp; conditions
+setup.registration.ifalreadyregistered=If you already have your e-mail and password registered at yetu, you can go to step 3 directly. Just click "next".
+
+setup.registration.error.firstname=Please use only characters for your first name.
```

Comment:
Some names include other characters like `-`. Please use only `characters` does not make much sense. Which characters? Also, validation on a First name / last name is unnecessary. We should not need such a field.